conductr_cli/test/test_sandbox_run.py

```diff
@@ -40,6 +40,26 @@ def default_general_args(self, container_name):
 
     container_names = ['cond-0']
 
+    class DummyFeature1:
+        def __init__(self, version_args, image_version, offline_mode):
+            return
+
+        def configure(self, args, run_result):
+            return
+
+        def start(self):
+            return []
+
+    class DummyFeature2:
+        def __init__(self, version_args, image_version, offline_mode):
+            return
+
+        def configure(self, args, run_result):
+            return
+
+        def start(self):
+            return []
+
```

Comment:
I can't seem to find the usage of `DummyFeature1` and `DummyFeature2`.

If they are meant to be used to assert that `configure` and `start` has been called, perhaps can be replaced with `MagicMock`, i.e.:

```
def create_dummy_features():
  feature = MagicMock()
  feature.configure = MagicMock()
  feature.start = MagicMock()
  return feature

feature_1 = self.create_dummy_features()
feature_2 = self.create_dummy_features()
features = [feature_1, feature_2] # use this as the return_value of mock_collect_features

feature_1.configure.assert_called_once_with(...)
feature_1.start.assert_called_once_with(...)
```

If they are indeed not used, can we remove them please?